routes/index.js

```diff
@@ -38,9 +38,10 @@ router.post("/", async(req, res) => {
 
     if (
       usersArr.find((user) => {
-        if (user.name === usrnm && user.room === room) return true;
+        if (currentUserData.usrnm === user.name && currentUserData.room === user.room) return true;
```

Comment:
Is [this](https://github.com/opencodeiiita/Geek-Chat/pull/259/commits/64e475bdb63ab60fdc7d13e2a2b34f40256a56b8#diff-103b73514ced7155e7e33849be2ab49fcbf55e6ff018e0e5356a82d74e6c7e09R33) not correct?

Reply:
it is correct, did'nt cause any problems as far as I tested